README.md

```diff
@@ -73,29 +128,27 @@ Then point your favourite browser to [http://localhost:3000/](http://localhost:3
 
 ### SlotPicker
 
-Used for choose and choose-date-and-time page. 
+Used for choose and choose-date-and-time page. This is included via [Bower](http://bower.io) and commited to this code base.
```

Comment:
Might be worth explicitly using the word `vendored` to convey to nerds that that's intended, if you see what I mean?